vmdb/app/presenters/menu/section.rb

```diff
@@ -8,6 +8,15 @@ def features
       Array(items).collect(&:feature).compact
     end
 
+    def visible?(userid)
+      store = Vmdb::PermissionStores.instance
+      tab   = Menu::Manager.tab_features_by_id(id)
+      auth  = store.can?(id) && User.current_user.role_allows_any?(:identifiers => tab)
```

Comment:
also `tab   = Menu::Manager.tab_features_by_id(id)` where `id` is id of the section actually returns `features` of the section if I am reading it right